realtime.go

```diff
@@ -99,16 +99,37 @@ type StopTimeEvent struct {
 }
 
 type VehicleID struct {
-	ID           string
-	Label        string
-	LicencePlate string
+	ID           *string
+	Label        *string
+	LicencePlate *string
 }
 
+type Position = gtfsrt.Position
+type CurrentStatus = gtfsrt.VehiclePosition_VehicleStopStatus
+type CongestionLevel = gtfsrt.VehiclePosition_CongestionLevel
+type OccupancyStatus = gtfsrt.VehiclePosition_OccupancyStatus
+
 type Vehicle struct {
 	ID *VehicleID
 
 	Trip *Trip
 
+	Position *Position
```

Comment:
Is there any benefit right now to creating a non-proto-generate `Position` struct and mapping accordingly? Maybe not?

Reply:
Good question, I couldn't think of a specific reason to do it for now so I just followed the existing pattern of type aliasing some existing gtfsrt proto types (e.g., how this was done for some enum types). That being said, I am happy to create a new struct to further decouple this library from the proto types if you think that would be cleaner.

Reply:
I think for the moment we can submit as-is, and if wanted later change it.